subsys/debug/coredump/coredump_internal.h

```diff
@@ -9,23 +9,6 @@
 
 #include <zephyr/toolchain.h>
 
-/**
- * @cond INTERNAL_HIDDEN
```

Comment:
Please do not remove the `@cond` line as it is a pair to the `@endcond` below.

Reply:
Done. Thanks!